advnote132/README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-   <img src="Arm_logo_blue_150MN.png" />
+   <img src="Arm_logo_blue_RGB.svg" />
```

Comment:
The advnote132, bsabi32 and dbgoverl32 documents are missing svgs. I'm guessing because they weren't merged yet when the first version of this patch were created.

Reply:
Oops! I wasn't awake yesterday. I remembered to delete the bitmaps from those three directories, but not to add the SVG in its place.